fggs/semirings.py

```diff
@@ -91,17 +92,19 @@ class RealSemiring(Semiring):
     def from_int(self, n: Union[int, torch.Tensor]):
         return torch.as_tensor(n, dtype=self.dtype, device=self.device)
     
-    add = staticmethod(torch.add) # type: ignore
     sum = staticmethod(torch.sum) # type: ignore
-    
+
+    @staticmethod
+    def add(x, y):
```

Comment:
Why no types on arguments?

Reply:
Sorry, I didn't realize that overridden methods don't inherit types from the parent class. Fixing in https://github.com/diprism/fggs/pull/163/commits/5f87f66b10ea6cf13cb69831f02c0af28800fb2c